rllib/evaluation/env_runner_v2.py

```diff
@@ -79,6 +99,27 @@ def get(self):
             "mean_env_render_ms": self.env_render_time * factor,
         }
 
+    def _get_ema(self):
+        # In EMA mode, simply ms -> sec.
```

Comment:
better: `sec -> ms`?

Or: "In EMA mode, stats are already (exponentially) averaged, hence we only need to do the sec -> ms conversion here."

Reply:
that's right. took your comment :)